src/components/Slider/index.tsx

```diff
@@ -76,36 +82,41 @@ const StyledBorderWrapper = styled(Box)`
   }
 `
 
-const Slider = ({ children, ...props }) => {
-  const scrollRef = useRef(null)
-  let intervalLeft
-  let intervalRight
+export const Item: FunctionComponent<{ as?: string | ElementType<unknown> }> =
+  ({ as, ...props }) => (
+    <StyledBorderWrapper as={as} {...props} draggable="true" />
+  )
+
+Item.propTypes = {
+  as: PropTypes.string,
+}
+
+type SliderType = FunctionComponent & {
+  Item: typeof Item
+}
+
+const Slider: SliderType = ({ children, ...props }) => {
+  const scrollRef = useRef<HTMLDivElement>(null)
+  let intervalLeft: ReturnType<typeof setInterval>
+  let intervalRight: ReturnType<typeof setInterval>
 
   const handleScrollRight = () => {
-    if (scrollRef.current) {
-      if (scrollRef.current.scrollTo) {
-        intervalRight = setInterval(() => {
-          scrollRef.current.scrollTo(scrollRef.current.scrollLeft - 25, 0)
-        }, 30)
-      }
+    if (scrollRef.current?.scrollTo) {
+      intervalRight = setInterval(() => {
+        scrollRef.current?.scrollTo?.(scrollRef.current?.scrollLeft - 25, 0)
+      }, 30)
     }
   }
   const handleScrollLeft = () => {
-    if (scrollRef.current) {
-      if (scrollRef.current.scrollTo) {
-        intervalLeft = setInterval(() => {
-          scrollRef.current.scrollTo(scrollRef.current.scrollLeft + 25, 0)
-        }, 30)
-      }
+    if (scrollRef.current?.scrollTo) {
+      intervalLeft = setInterval(() => {
+        scrollRef.current?.scrollTo(scrollRef.current?.scrollLeft + 25, 0)
```

Comment:
```suggestion
    if (scrollRef.current?.scrollTo) {
      intervalLeft = setInterval(() => {
        scrollRef.current.scrollTo(scrollRef.current?.scrollLeft + 25, 0)
```

We already check if the value exists